tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2864,6 +2952,17 @@ mvrf/test_mgmtvrf.py::TestReboot::test_warmboot:
     conditions:
       - "topo_name in ['dualtor', 'dualtor-56', 'dualtor-120', 'dualtor-aa', 'dualtor-aa-56', 'dualtor-aa-64-breakout']"
 
+#######################################
+#####           mx              #####
+#######################################
+
+mx:
+  skip:
+    reason: "Test is skipped for Mellanox SN5640 platform,dualtorand for  Arista-7060X6-64PE-B-* TH5 SKUs"
+    conditions:
+      - "topo_type !in ['m0', 'mx', 'm1']"
+      -  "hwsku in ['Arista-7060X6-64PE-B-C448O16','Arista-7060X6-64PE-B-C512S2','Arista-7050CX3-32S-C32','Arista-7050CX3-32S-D48C8','Arista-7260CX3-D108C8','Arista-7260CX3-C64','Mellanox-SN4700-V64','Arista-7050CX3-32C-C32','Cisco-8101-O8C48','Cisco-8101-V64','Cisco-8101C01-C32','Mellanox-SN5640-C512S2','Mellanox-SN5640-C448O16']"
```

Comment:
`- "hwsku in ['Arista-7060X6-64PE-B-C448O16','Arista-7060X6-64PE-B-C512S2','Mellanox-SN4700-V64','Mellanox-SN5640-C512S2','Mellanox-SN5640-C448O16']"`